packages/rolldown/tests/fixtures/plugin/context/cycle-load-error/_config.ts

```diff
@@ -8,19 +8,19 @@ export default defineTest({
     plugins: [
       {
         name: 'test-plugin-context',
-        onLog: (level, log) => {
```

Comment:
`onLog` in plugin does not prevent the log from being written to `stderr`, but top level `onLog` does.